### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by michaelamponsah**
# Hello ☺️,

### In this PR, the following requirements have been implemented.

- [x]  Created a UI for the School Library Application

lib

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    print 'ID of person: '
+    person_id = gets.chomp
+    puts 'Rentals: '
+    list_rental_by_id(person_id.to_i)
+  end
+
+  def list_rental_by_id(person_id)
+    @rentals.each do |rental|
+      if rental.person.id == person_id
+        puts "\nDate: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author}"
```
Comment (jerryowusu) [inline app.rb:47]: In  trying to list all rentals for a giving person the actual basic UI keeps it simple by asking you to type the id of the person. However, you try to populate the list before asking to type the id which is fine but there is no space between the request for the id and the last element of the list as shown below. This makes it difficult to read and not the best standard UI practice. Kindly update your UI to match the actual one in order to make it simple and easy to read. :pray: 

### Your UI

Screenshot from 2023-03-11 16-16-38



### Actual UI

Screenshot from 2023-03-11 16-09-06

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @michaelamponsah , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

no-we-were-so-close-pauline-turner-brooks


## Highlights
Nice work so far :heavy_check_mark: 
Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (michaelamponsah) [reply app.rb:47]: @jerryowusu, many thanks for your review and feedback. I'd work on it right away!

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @michaelamponsah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- PR is professional :+1: 
- Console UI app works as expected :+1: 
- Linter checks passed :heavy_check_mark: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by michaelamponsah** [comment]
@FrederickMih, Many thanks for your review! Much appreciated.

**Pull request merged**